WebDriverAgentTests/IntegrationApp/Classes/FBInvisibleParentScrollingController.m

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "FBInvisibleParentScrollingController.h"
+#import "FBTableDataSource.h"
+
+static const CGFloat FBSubviewHeight = 40.0;
+
+@interface FBInvisibleParentScrollingController ()
+@property (weak, nonatomic) IBOutlet UIScrollView *scrollView;
+@property (strong, nonatomic) IBOutlet FBTableDataSource *dataSource;
+
+@end
+@implementation FBInvisibleParentScrollingController
+
+- (void)viewDidLoad {
```

Comment:
I don't like this. This is very screen size specific and wouldn't test what we want to test. Please drop all `FBInvisibleParentScrollingController` and`FBInvisibleParentScrollingTests`. I'll add simple test for it.